Add profile fields on signup page

PatrickTUy opened the pull request:
### PR Description
This PR added more attributes  when signing up.
### Description of tasks that were expected to be completed
add more attributes on create user mutation
### Functionality
when a user sign ups they add more attributes like last name, first name, gender and date of birth.
### How has this been tested?
checkout this PR  [link](https://github.com/atlp-rwanda/atlp-pulse-fn/pull/138) on front end 
### PR Checklist:
- [x] Add more attributes when signing up.

### Track PR
[#DP-120](https://trello.com/c/Ym1TsdmM/120-signup-page)
### Screenshots (If appropriate)
(Images)

Procech20 commented:
I deployed a review app for you to try out this pull request 👇🏽

Heroku App: https://devpulse-bn-review-app-42.herokuapp.com/


<hidden purpose="for-rewritable-pr-comment-action-use" value="Deploy"></hidden>

Himbaza123h reviewed (approved):
Great

PatrickTUy committed:
feat(DP-123):Create backend review apps (#38) (#40)

- create review app for opened, re-opened and synched PR
- update the app if there're new changes added to the PR
- close app if PR is merged or closed

[#Delivers DP-123]

added profile fields on signup

fixed profile issues

Co-authored-by: Prophete ISINGIZWE <75941936+Procech20@users.noreply.github.com>
